packages/node-image-non-compute-common/base.packages

```diff
@@ -123,11 +125,15 @@ squid=4.17-150000.5.32.1
 strace=5.3-1.44
 sudo=1.9.5p2-150300.3.6.1
 sysstat=12.0.2-3.33.1
-systemtap=4.2-3.3.1
+systemd-coredump=246.16-150300.7.51.1
+systemd-lang=246.16-150300.7.51.1
+systemd-sysvinit=246.16-150300.7.51.1
 systemd=246.16-150300.7.51.1
+systemtap=4.2-3.3.1
```

Comment:
Removing this.. didn't mean to add it.

The diff that this rebuild makes does bring in some other changes.. Pete, Heemstra, and I are reviewing those.